data/avgi/map.txt

```diff
@@ -0,0 +1,5276 @@
+			sprite planet/station-outpost-a3
+				scale 0.5
+			distance 240
+			period 42
+		object
+			sprite planet/debris1
+				scale 0.5
+			distance 240
+			period 42
+		object
+			sprite planet/station-weapon-a2
+				scale 0.5
+			distance 180
+			period 42
+			offset 180
+		object
+			sprite planet/debris1
+				scale 0.5
+			distance 180
+			period 42
+			offset 180
+		object
+			sprite planet/station-reflector-a1
+				scale 0.25
+			distance 120
+			period 42
+		object
+			sprite planet/station-reflector-a0
+				scale 0.25
+			distance 135
+			period 42
+			offset 40
+		object
+			sprite planet/station-reflector-a1
+				scale 0.25
+			distance 125
+			period 42
+			offset 80
+		object
+			sprite planet/station-reflector-a0
+				scale 0.25
+			distance 115
+			period 42
+			offset 120
+		object
+			sprite planet/station-reflector-a1
+				scale 0.25
+			distance 120
+			period 42
+			offset 200
+		object
+			sprite planet/station-reflector-a0
+				scale 0.25
+			distance 125
+			period 42
+			offset 280
+		object
+			sprite planet/debris1
+				scale 0.5
+			distance 140
+			period 42
+			offset 80
+		object
+			sprite planet/debris0
+				scale 0.25
+			distance 120
+			period 42
+			offset 205
+		object
+			sprite planet/debris1
+				scale 0.5
+			distance 140
+			period 42
+			offset 290
+	object
+		sprite planet/gas3-b
+		distance 7258.824
+		period 5274.096
+		object
+			sprite planet/callisto-b
+			distance 333.434
+			period 149.109
+		object Anitlos
+			sprite planet/europa-b
+			distance 457.248
+			period 474.342
+		object
+			sprite planet/ice8-b
+			distance 576
+			period 522.558
+	object
+		sprite planet/station-cylinder-broken
+			scale 0.5
+		distance 7258.824
+		period 5274.096
+		offset 10
+	object
+		sprite planet/station-cylinder-incomplete
+			scale 0.5
+		distance 7258.824
+		period 5274.096
+		offset -10
+	object "Nisos Cylinder"
+		sprite planet/station-cylinder
+			scale 0.5
+		distance 7258.824
+		period 5274.096
+		offset -15
+	object
+		sprite planet/station-cylinder-broken
+			scale 0.5
+		distance 7258.824
+		period 5274.096
+		offset -20
+	object
+		sprite planet/station-cylinder-broken
+			scale 0.5
+		distance 7258.824
+		period 5274.096
+		offset -25
+	object
+		sprite planet/gas2
+		distance 10719.178
+		period 9464.352
+		object
+			sprite planet/station-outpost-a0
+				scale 0.5
+			distance 340
+			period 44
+			offset 180
+		object Entantel
+			sprite planet/ice8-b
+			distance 267.286
+			period 44
+	object
+		sprite planet/gas1-b
+		distance 15027.451
+		period 15710.016
+		object
+			sprite planet/telescope-a1
+				scale 0.5
+			distance 423.741
+			period 60
+		object
+			sprite planet/ice8-b
+			distance 278.66
+			period 430.255
+
+system Clip
+	pos -508.501 711.522
+	government Avgi
+	attributes avgi twilight
+	habitable 625
+	belt 1750 4
+	haze _menu/haze-twilight
+	link Anax
+	asteroids "large metal" 1 3.181
+	fleet "Avgi Scouts" 36000
+	fleet "Small Aberrant" 2300
+	fleet "Large Aberrant" 3900
```

Comment:
Sail (Araio, spaceport destroyed) and Anax (Aviskir, spaceport active) have trade prices. Soar, Mellow, Thread, Clip, and Wake do not. I assume this is just something you haven't gotten around to, but in case it's meant to represent there not being coordinated enough supply lines to take tons of food to another planet it's worth noting.